.vscode/settings.json

```diff
@@ -13,7 +13,7 @@
   ],
   "python.formatting.provider": "yapf",
   "python.venvPath": "${workspaceFolder}/.venv/",
-  "python.pythonPath": "${workspaceFolder}/.venv/Scripts/python.exe",
+  "python.pythonPath": ".venv\\Scripts\\python.exe",
```

Comment:
Can you make this a windows specific settings(I think if you do
```json
"windows": {
  "python.pythonPath": ".venv\\Scripts\\python.exe"
}
```

Reply:
vscode doesn't like the windows key, not sure if this is possible. Do you have any docs on this? Would be really nice to have OS specific settings.